app/graphql/albums/middlewares.js

```diff
@@ -0,0 +1,33 @@
+const errors = require('../../errors'),
```

Comment:
it's interesting what you did. We see the concept of middleware as a reusable component between endpoints. I don't see that in this usecase. Anyways, you pre validation isolated from the resolver it's nice! I'd just move it to another place, like an interactor

Reply:
ok!